packages/addon/app/helpers/ensure-safe-component.js

```diff
@@ -0,0 +1,7 @@
+import { ensureSafeComponent } from '@embroider/addon';
+import Helper from '@ember/component/helper';
+export default class extends Helper {
+  compute([value]) {
+    return ensureSafeComponent(value, this);
+  }
+}
```

Comment:
I agree that would be more conventional, and I should probably make the change you suggest. This started out as a functional helper, in which case there really was basically no implementation to worry about. But then it became a class-based helper, and now has a bunch of syntax that *might* in theory be problematic for an app.

But I don't think this actually breaks under any real conditions. The thing about the app folder isn't that it doesn't get transpiled, it's that it gets transpiled under the app's babel config and not the addon's. But what is the minimum level of Javascript features we can assume will work in the app? We already know that the app *must* support ES modules because people put reexport syntax in the app tree. Are there really any apps that support ES modules but not class syntax or array destructuring? Probably not.